network/server_test.go

```diff
@@ -14,6 +14,21 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func generateTestKey(t *testing.T) (crypto.PrivKey, string) {
+	t.Helper()
+
+	dir, err := ioutil.TempDir(os.TempDir(), "")
+	assert.NoError(t, err)
+	key, err := ReadLibp2pKey(dir)
+	assert.NoError(t, err)
+	t.Cleanup(func() {
+		// remove directory after test is done
+		assert.NoError(t, os.RemoveAll(dir))
+	})
+
+	return key, dir
+}
+
```

Comment:
@Kourin1996 
Do you think we should maybe move this out to the `helper` package, or leave it here?

Just a thought

Reply:
I tried but there will be a cyclic dependency (because ReadLibp2pKey is defined in network package)